wavedump/src/wavedump.c

```diff
@@ -2039,20 +2195,38 @@ int main(int argc, char *argv[])
         exit(1);
     }
 
+    // for (i = 0; i < 16; i++) {
+    //     ret = CAEN_DGTZ_SetChannelDCOffset(handle, i, 0x7FFF);
+    // 	if (ret) {
+    //         fprintf(stderr, "failed to set DC offset for one of the channels\n");
+    //     }
+    // }
+
+
+
+    /* Set the channels to trigger on themselves */
+    // ret = CAEN_DGTZ_SetChannelSelfTrigger(handle, CAEN_DGTZ_TRGMODE_ACQ_ONLY, 0xffff);
+    // 
+    // if (ret) {
+    //     fprintf(stderr, "failed to set self trigger!\n");
+    //     exit(1);
+    // }

```

Comment:
Might want to add these in a comment above the previous code, saying "This is how you should be able to do it using the CAEN API, but we found it didn't work so we instead set the registers directly"